recipe/menu-v2.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "$id": "https://schemas.conda.io/menuinst-1.schema.json",
```

Comment:
Shouldn't the id be menuninst-2.schema?

Reply:
No, this is the correct value. See here: https://conda.github.io/menuinst/reference/#menuinst._schema.MenuInstSchema.id_